src/Presentation/UiBuilder.php

```diff
@@ -52,32 +61,46 @@ private function parseQuery( string $searchQuery ): Query {
 	/**
 	 * @return array<array<string, string>>
 	 */
-	private function buildInstancesViewModel(): array {
-		// TODO: Get instances from config
-		// TODO: Get currently selected instance from query
-		return [
+	private function buildInstancesViewModel( ?PropertyId $instancePropertyId, ?ItemId $instanceItemId ): array {
```

Comment:
instancePropertyId parameter is unused (and also does not make sense since you can $this->config->getInstanceOfId())